identityClient/api.go

```diff
@@ -207,3 +207,15 @@ type CompleteChallengeRequest struct {
 	SignedChallenge string `json:"signed_challenge"` // signed challenge in the form of "{challenge}{timestamp}"
 	SignedTime      int64  `json:"sign_time"`        // time challenge was signed
 }
+
+type LDAPCache struct {
+	ID                 string              `json:"id,pk"`
+	LDAPID             string              `json:"ldap_id"`
+	DN                 string              `json:"dn"`
+	RdnAttributeName   string              `json:"rdn_attribute_name"`
```

Comment:
and comments for struct fields that are not apparent to non LDAP experts. If possible for struct field names try to not go with contractions (obviously the json tag has to for compatibility with api's, but this is code we want to be compatible or at least more readable for humans)

Reply:
I actually don't know what many of these fields are... they are names from the Java LDAP objects, I just cached the values with the same names used... I _think_ dn is 'domain name' for example. I'll see if some quick googling will reveal what these things are.